src/main/java/com/google/devtools/build/lib/runtime/commands/CleanCommand.java

```diff
@@ -191,6 +227,7 @@ private void actuallyClean(CommandEnvironment env,
     if (cleanOptions.expunge || cleanOptions.expunge_async) {
       throw new ShutdownBlazeServerException(0, ShutdownMethod.EXPUNGE);
     }
+    System.gc();
```

Comment:
We noted specifically that workspace.clearCaches() removed nearly all object utilization, and that a gc invocation was definitive in demonstrating that nearly all of bazel's (vm) persistent heap usage was tied up in the cache.  Meaningless in the expunge case, it was just an opportunity to show it had nothing else up its sleeves.  I'm fine with removing it, it just felt right.